### PR TITLE
Document SCL's standard grooming practice

### PR DESCRIPTION
**Pull request by timothysc**
/assign @detiber @neolit123 @roberthbailey 
/cc @BenTheElder @spiffxp @kubernetes/sig-cluster-lifecycle-pr-reviews

### CODE REVIEW

File: sig-cluster-lifecycle/grooming.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+This document is an overview of the practices that are currently followed and any feedback, or refinement, is solicited. 
+
+## Process 
+
+### Planning a milestone
+For every sub-project under sig-cluster lifecycle, we highly recommend holding a planning session prior to beginning a new milestone.  The purpose of planning is to outline a set of high level deliverables that the community has decided that they want to work on for this upcomming milestone.  
+
+### Creating milestones
+Now that you have a plan, you're about to embark upon a new journey, let's call this journey `1.X` on a fresh repo.  The first step is to create two milestones (`1.X`, and `Next`).  The reason for the `Next` milestone is simply to create a placeholder to segregate work items out of the current milestone.  We typically tr to stick with just two milestones, because we have found over time that this simplfies execution and keeps us focused on trying to deliver for the current milestone.
```
Comment (neolit123) [inline sig-cluster-lifecycle/grooming.md]: `We typically try` <- missing `y` in `try`

File: sig-cluster-lifecycle/grooming.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+## Overview 
+
+The purpose of this document is to outline the current backlog grooming practices, or process, that are employed across several sub-projects under SIG Cluster Lifecycle.  By providing a semi-consistent set of best practices it allows developers to work across sub-projects fairly easily.  More importantly it signals direction and priority to developers who may be new to the sub-projects.  
+
+This document is an overview of the practices that are currently followed and any feedback, or refinement, is solicited. 
+
+## Process 
+
+### Planning a milestone
+For every sub-project under sig-cluster lifecycle, we highly recommend holding a planning session prior to beginning a new milestone.  The purpose of planning is to outline a set of high level deliverables that the community has decided that they want to work on for this upcomming milestone.
```
Comment (neolit123) [inline sig-cluster-lifecycle/grooming.md]: as the CI suggests `upcomming` -> `upcoming`

File: sig-cluster-lifecycle/grooming.md (hunk 1 above)
Comment (timothysc) [reply sig-cluster-lifecycle/grooming.md]: Fixed

File: sig-cluster-lifecycle/grooming.md (hunk 2 above)
Comment (timothysc) [reply sig-cluster-lifecycle/grooming.md]: Fixed.

**Comment by BenTheElder** [comment]
Thank you for documenting this! :smile:

**Comment by fabriziopandini** [comment]
@timothysc Thanks!
/approve
/lgtm

**Comment by timothysc** [comment]
comments addressed, canceling hold and folks can always PR to refine. 
/hold cancel

**Pull request merged**